### PR TITLE
Feature/DTGB-501: Added styling for person detail page + refactor of boxes in detail layouts.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Review by delrueba — CHANGES_REQUESTED** [review]
Can you add the li font-size fix for detail layout?

**Review by delrueba — APPROVED** [review]

File: styleguide/components/31-molecules/contact-details/contact-details.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            '<i class="icon-fax" aria-hidden="true"></i><a href="tel:+32 9 266 50 19" alt="Person fax number">09 266 50 19</a> (Fax)',
+            '<i class="icon-envelope" aria-hidden="true"></i><a href="mailto:daniel.termont@stad.gent" alt="Person mail link">daniel.termont@stad.gent</a>'
+          ]
+        }
+      %}
+    </div>
+
+    <div class="contact-details-column">
+      {% include '@image' with {
+        'src': 'https://via.placeholder.com/280x280',
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/contact-details/contact-details.twig]: ratio mee vermelden

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/contact-details/contact-details.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/contact-details/contact-details.twig]: Fixed.

File: styleguide/components/41-organisms/timeline/timeline.twig [hunk 2]
```diff
@@ -172,6 +172,101 @@
...
     </div>
+    <div class="timeline-slot">
+      <dt>1,4 till 1,6 milion years ago</dt>
+      <dd>
+        <div class="timeline-slot-header">
+          <h3 class="timeline-slot-title"><button aria-expanded="true" aria-controls="timeline-slot-content--7" class="accordion--button">Title</button></h3>
+          <span class="timeline-slot-date">On a thursday</span>
+          <div class="image-gallery-wrapper">
+            {% include '@image-gallery' with {
+              "src": "https://loremflickr.com/704/440/ghent?random=14",
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/timeline/timeline.twig]: Is het niet beter om hier ook de 800x500 rule te hanteren? Tevens in organisms zou ik geen loremflickr gebruiken omdat je hierdoor de ratio docs mist.

File: styleguide/components/31-molecules/contact-details/contact-details.twig [hunk 3]
```diff
@@ -22,8 +22,8 @@
 
     <div class="contact-details-column">
       {% include '@image' with {
-        'src': 'https://via.placeholder.com/280x280',
-      'alt_text': 'Person avatar',
+        'src': 'https://via.placeholder.com/280x280&text=1:1+(280x280',
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/contact-details/contact-details.twig]: ) op het einde

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/timeline/timeline.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/41-organisms/timeline/timeline.twig]: Aangepast.

File: styleguide/components/31-molecules/contact-details/contact-details.twig (hunk 3 above)
Comment (meirege) [reply styleguide/components/31-molecules/contact-details/contact-details.twig]: Fixed.

File: styleguide/components/41-organisms/timeline/timeline.twig [hunk 4]
```diff
@@ -11,17 +11,17 @@
             {% include '@image-gallery--multiple' with {
               "items": [
                 {
-                  "src": "https://loremflickr.com/704/440/ghent?random=1",
+                  "src": "https://loremflickr.com/800/500/ghent?random=1",
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/timeline/timeline.twig]: Dit is een organisms, best geen content voorzien: https://via.placeholder.com/800x500&text=8:5+(800x500)

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/timeline/timeline.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/41-organisms/timeline/timeline.twig]: Aangepast.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**